### ISSUE TITLE
Tabs: Tabs title not visible in version-1.3.0

### ISSUE THREAD

**Issue by shyamal890** [driftyco/ionic#6389]
Tabs' title are no longer visible in 1.3.0, works with 1.2.4

Seems like addition of `<i class="icon"></i>` in https://github.com/driftyco/ionic/commit/fbae1aba is forcing title to new line, is hidden due to `overflow:hidden`. I wonder how this commit made it without tests or is it a breaking change and documentation is not yet updated?

Here is an Example Plunker: https://plnkr.co/edit/P71TcR?p=preview

**Ionic Version**: 1.3.0
**Ionic Info**:
Cordova CLI: 6.1.1
Ionic Version: 1.3.0
Ionic CLI Version: 1.7.14
Ionic App Lib Version: 0.7.0
OS: Windows 7 SP1
Node Version: v4.4.3

**Comment by calebeaires** [comment]
This fix for now the problem! 

```
// just when not using icons on tabs
.tab-title {
    font-size: 14px;
    margin-top: -34px !important;
    display: block;
}

```

**Comment by danbucholtz** [comment]
This issue is resolved in the nightly build and will be in 1.3.1.

Thanks,
Dan
